src/it/warResources/verify.groovy

```diff
@@ -20,8 +20,7 @@ globalConfigHelp  = new File(basedir, 'target/war-resources-it/help-globalConfig
 // code uses plexus not maven so escapes are not supported.
 assert globalConfigHelp.contains("\\\${prop1\\} hello");
 assert globalConfigHelp.contains("\\@prop2\\@ goodbye");
-// code claims to add System properties but this fails
-// assert globalConfigHelp.contains("\\@java.class.version\\@ 55.0");
+assert globalConfigHelp.contains("\\@java.vendor.url\\@ https://");
```

Comment:
System properties now work (which was expected before from the code - but did not actually work)